django/contrib/postgres/aggregates/general.py

```diff
@@ -1,14 +1,63 @@
 from django.contrib.postgres.fields import JSONField
 from django.db.models.aggregates import Aggregate
+from django.db.models.expressions import F, OrderBy
 
 __all__ = [
     'ArrayAgg', 'BitAnd', 'BitOr', 'BoolAnd', 'BoolOr', 'JSONBAgg', 'StringAgg',
 ]
 
 
-class ArrayAgg(Aggregate):
+class OrderableAgg(Aggregate):
+    template = "%(function)s(%(expressions)s %(ordering)s)"
```

Comment:
Use single quotes (unless a string contains a single quote) as described in [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).